pkgs/development/python-modules/formulaic/default.nix

```diff
@@ -0,0 +1,68 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytestCheckHook
+, hatchling
+, hatch-vcs
+, git
+, astor
+, interface-meta
+, numpy
+, pandas
+, scipy
+, sympy
+, wrapt
+, typing-extensions
+}:
+
+buildPythonPackage rec {
+  pname = "formulaic";
+  version = "0.5.2";
+
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "matthewwardrop";
+    repo = "formulaic";
+    rev = "v${version}";
+    sha256 = "1ql41wrs198gxkdg0agg5pibsffigf0s8sj1x90gm0ydz9mb8f5x";
+    leaveDotGit = true;
+  };
+
+  SETUPTOOLS_SCM_PRETEND_VERSION = version;
+
+  nativeBuildInputs = [
+    git
+  ];
+
+  buildInputs = [
```

Comment:
```suggestion
```
Do we really need git? Also hatchling is run at build time so we need it native.

Reply:
With SETUPTOOLS_SCM_PRETEND_VERSION set, git is no longer necessary.

Thanks.